pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractTopic.java

```diff
@@ -439,6 +439,9 @@ private PublishRate publishRateInBroker(ServiceConfiguration config) {
     }
 
     protected boolean isProducersExceeded() {
+        if (isSystemTopic()) {
```

Comment:
Hmm, shall we need to skip the replicator also?

Reply:
updated.